src/frontend/src/optimizer/rule/distinct_agg_rule.rs

```diff
@@ -281,13 +281,17 @@ impl DistinctAggRule {
                     AggKind::BitAnd
                     | AggKind::BitOr
                     | AggKind::BitXor
+                    | AggKind::BoolAnd
+                    | AggKind::BoolOr
                     | AggKind::Min
                     | AggKind::Max
                     | AggKind::Sum
                     | AggKind::Sum0
                     | AggKind::Avg
                     | AggKind::StringAgg
                     | AggKind::ArrayAgg
+                    | AggKind::JsonbAgg
+                    | AggKind::JsonbObjectAgg
                     | AggKind::FirstValue
                     | AggKind::StddevPop
                     | AggKind::StddevSamp
```

Comment:
Seems the "partial to global" logic is duplicated to `partial_to_total_agg_call` in `generic::Agg` node. Maybe we should extract a common function to ensure consistency later.